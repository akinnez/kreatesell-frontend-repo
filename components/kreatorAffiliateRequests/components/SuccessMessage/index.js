import {Typography} from 'antd';
import styles from './index.module.scss';

const {Text} = Typography;

const SuccessMessage = () => (
	<section className={styles.content}>
		<p>
			<Text>Report Successfully Sent</Text>
		</p>
		<p>
			<Text>
				We would review it and if the affiliate is found guilty, they
				would no longer have access to your products.
			</Text>
		</p>
	</section>
);

export default SuccessMessage;
