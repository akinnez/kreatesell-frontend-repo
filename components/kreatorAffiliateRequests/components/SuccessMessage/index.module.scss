@use 'public/css/colors';

.content {
	margin-bottom: 40px;

	p {
		margin-bottom: 0;
		text-align: center;

		&:first-child [class='ant-typography'] {
			font-weight: 600;
			font-size: 30px;
			color: colors.$black-100;
		}

		&:last-child [class='ant-typography'] {
			font-weight: 400;
			font-size: 16px;
			color: colors.$gray-200;
		}
	}
}
