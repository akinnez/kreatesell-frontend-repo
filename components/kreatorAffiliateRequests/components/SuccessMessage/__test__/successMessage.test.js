import SuccessMessage from '..';
import renderer from 'react-test-renderer';

it('renders a snapshot of <SuccessMessage />', () => {
	const message = renderer.create(<SuccessMessage />).toJSON();
	expect(message).toMatchSnapshot();
});
