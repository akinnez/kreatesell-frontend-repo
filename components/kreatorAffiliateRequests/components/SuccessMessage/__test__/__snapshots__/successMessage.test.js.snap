// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <SuccessMessage /> 1`] = `
<section
  className="content"
>
  <p>
    <span
      className="ant-typography"
      onClick={null}
      style={
        Object {
          "WebkitLineClamp": undefined,
        }
      }
    >
      Report Successfully Sent
    </span>
  </p>
  <p>
    <span
      className="ant-typography"
      onClick={null}
      style={
        Object {
          "WebkitLineClamp": undefined,
        }
      }
    >
      We would review it and if the affiliate is found guilty, they would no longer have access to your products.
    </span>
  </p>
</section>
`;
